Formula/v/veccore.rb

```diff
@@ -0,0 +1,34 @@
+class Veccore < Formula
+  desc "C++ Library for Portable SIMD Vectorization"
+  homepage "https://github.com/root-project/veccore"
+  url "https://github.com/root-project/veccore/archive/refs/tags/v0.8.2.tar.gz"
+  sha256 "1268bca92acf00acd9775f1e79a2da7b1d902733d17e283e0dd5e02c41ac9666"
+  license "Apache-2.0"
+  head "https://github.com/root-project/veccore.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "vc"
```

Comment:
does it require vc to run?

Reply:
optionally,
see https://github.com/root-project/veccore

Additionally, you will need CMake 3.16 or later, and you may want to install a SIMD library such as
    [Vc](https://github.com/VcDevel/Vc) (version 1.4 or later)
    [UME::SIMD](https://github.com/edanor/umesimd) (version 0.8.1 or later)
    [std::experimental::simd](https://gcc.gnu.org/gcc-11/changes.html#libstdcxx) (included in libstdc++ from GCC 11 or later)

Reply:
with vc is better

Reply:
got you, thanks!